### PR TITLE
Fix: content type application/x-www-form-urlencoded is not allowed

### PR DESCRIPTION
**Pull request by paring-chan**
# What

fixes `Unsupported Media Type` error for `application/x-www-form-urlencoded` content type

# Why

Without this, some of tools that integrates with misskey will be broken. (example: https://misskey.tools)  
I think this is caused by the default content type on POST request of axios

### CODE REVIEW

File: packages/backend/src/server/ServerService.ts [hunk 1]
```diff
@@ -70,6 +70,8 @@ export class ServerService {
 			});
 		}
 
+		fastify.register(fastifyFormBody);
```
Comment (syuilo) [inline packages/backend/src/server/ServerService.ts]: Maybe ApiServerService is better place

**Comment by HotoRas** [comment]
Note: this will fix #9448 if works as expected.
Checked on the test environment (dev build, server & domain provided by @pikokr, code on this pr) and worked quite well.

**Comment by tamaina** [comment]
p1.a9z.devに適用してみたが、 #9448 (Misskey Toolsのエラー)は治らなかった

**Comment by syuilo** [comment]
そもそもMisskeyAPIの仕様的に今までapplication/x-www-form-urlencodedってサポートしてたっけ？

**Comment by tamaina** [comment]
https://github.com/misskey-dev/misskey/pull/3740

こんなプルリクがあった

**Comment by syuilo** [comment]
see https://github.com/misskey-dev/misskey/pull/3740

**Pull request closed without merging**